### PR TITLE
Loading: add fading animations

### PR DESCRIPTION
**Pull request by Leopoldthecoder**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: packages/loading/src/directive.js [hunk 1]
```diff
@@ -36,37 +36,42 @@ exports.install = Vue => {
...
-        if (binding.modifiers.fullscreen && el.originalOverflow !== 'hidden') {
-          document.body.style.overflow = el.originalOverflow;
-        }
-        if (binding.modifiers.fullscreen || binding.modifiers.body) {
-          document.body.style.position = el.originalPosition;
-        } else {
-          el.style.position = el.originalPosition;
-        }
+        const destroyElement = function() {
+          el.mask.removeEventListener('transitionend', destroyElement);
```
Comment (QingWei-Li) [inline packages/loading/src/directive.js]: 话说这个是不是统一下写成个方法啥的，用的地方蛮多的，毕竟要处理各种前缀

Comment (Leopoldthecoder) [inline packages/loading/src/directive.js]: transitionend 应该不用前缀了

Comment (baiyaaaaa) [inline packages/loading/src/directive.js]: 只有 ie9 不支持 transitionend，这个需要判断一下吧

Comment (Leopoldthecoder) [inline packages/loading/src/directive.js]: @baiyaaaaa 已更新

**Pull request merged**